Source/DafnyCore/Resolver/Resolver.cs

```diff
@@ -464,7 +464,7 @@ enum ValuetypeVariety {
       if (Options.DisallowConstructorCaseWithoutParentheses) {
         rewriters.Add(new ConstructorWarning(reporter));
       }
-      rewriters.Add(new UselessOldLinter(reporter));
+      rewriters.Add(new UselessOldAndRustAssertLinter(reporter));
```

Comment:
Because you are combining seemingly two very different linters, and there might be case for other things like that in the future, could you rename this one to `LocalInter` ? `PrecedenceLinter` is everything but local so it can stay in its own class.

Reply:
"Local" -- nice way to make the distinction. I'll do that.